### DISCUSSION TITLE
Neverball Icon SVG version

### DISCUSSION THREAD

**Discussion by jfietkau** [Neverball/neverball#301] [Show and tell]
Hi all! I was unable to find a vector version of the Neverball logo/icon, so I made one myself. I took `dist/neverball_512.png` as a close inspiration and put together a vector version in the SVG format.

It's not pixel-perfect, but it's as close as I could get it. Swapping back and forth between them, I think my shadows are slightly less harsh.

If the Neverball project would like to use this, whether for platforms that natively support SVG (like the website) or for higher-resolution renders, I'm happy to give permission to use, modify and distribute this version in any way you see fit.

This file is already optimized for size.

![Neverball Vector](https://user-images.githubusercontent.com/1278511/183968402-6186a648-d88d-434a-a423-fa7624c163e4.svg)

**Comment by parasti** [comment]
That's cool. I have to mention, though, that we have an SVG file in the dist folder. It contains logos for both Neverputt and Neverball.

**Reply by jfietkau** [reply]
Oh, ha. I wonder how many times I missed that.

Looking at the file now, it seems like it doesn't have the original drop shadow, but apart from that it's complete. I'm experiencing some glitches in the shading near the top of the ball in eog though, I wonder what's going on there. It looks better in Firefox.

I'll note for posterity that my version is easier on the viewer's resources since I built it out of gradients and without using any blur effects. The existing SVG really makes the fans spin up if I zoom in and out.
